dashboards/Insight/Prometheus_Exporter_Status.json

```diff
@@ -4533,7 +4650,7 @@
       "yBucketBound": "auto"
     }
   ],
-  "refresh": "1m",
+  "refresh": "5s",
```

Comment:
Default refresh time is 1m. Five second is too often.

Reply:
Done